cleanslate/doma/templates/home.html

```diff
@@ -1,6 +1,6 @@
 {% extends "./base.html" %}
 
-{% block title %}<title>Calendar</title>{% endblock %}
+{% block title %}<title>Home</title>{% endblock %}
```

Comment:
Why change the title of the home page to 'calendar'?

Reply:
Errr idk man

Reply:
It's from master. Hold on.

Reply:
Wow. I'm a genius.